### PR TITLE
Adding license header constant in SiteRecovery spec

### PR DESCRIPTION
**Pull request by avneeshrai**
Change is to address the comment on PR: https://github.com/Azure/azure-sdk-for-net/pull/3194

This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 

### PR information
- [x] The title of the PR is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
- [x] If applicable, the PR references the bug/issue that it fixes.
- [x] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).

### Quality of Swagger
- [x] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
- [x] My spec meets the review criteria:
  - [x] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
  - [x] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
  - [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.

---
This change is [Reviewable](https://reviewable.io/reviews/azure/azure-rest-api-specs/1227)

### CODE REVIEW

File: arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json [hunk 1]
```diff
@@ -9194,12 +9190,7 @@
           "$ref": "#/definitions/FabricSpecificUpdateNetworkMappingInput"
         }
       ],
-      "properties": {
-        "InstanceType": {
-          "description": "The instance type.",
-          "type": "string"
-        }
-      },
+      "properties": {},
```
Comment (jhendrixMSFT) [inline arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json]: Is there a reason why this was left empty?

File: arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json [hunk 2]
```diff
@@ -6855,7 +6858,7 @@
       "description": "Input details specific to fabrics during Network Mapping.",
       "type": "object",
       "properties": {
-        "InstanceType": {
+        "instanceType": {
```
Comment (jhendrixMSFT) [inline arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json]: Did the service change or is this update to the swagger making it match the service?

Comment (avneeshrai) [inline arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json]: This update to swagger matches it to service.

File: arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json (hunk 1 above)
Comment (avneeshrai) [inline arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json]: @jhendrixMSFT model VmmToAzureUpdateNetworkMappingInput in service as of now doesn't have any value specific to this instance as of now and has been introduced to make it future extensible and consistent with other sibling models (we expect it to have some fields in near future).
Swagger changes just represent what the service reflects

Swagger and changes in swagger exa

**Comment by avneeshrai** [comment]
@jhendrixMSFT responded to your comments. Let me know if any other action item is on me or merge if it can be merged :)

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**